src/main/java/ultoi/util/UltoiException.java

```diff
@@ -0,0 +1,77 @@
+package ultoi.util;
+
+/**
+ * Represents the exceptions that could arise during the execution of  Ultoi bot.
+ */
+public class UltoiException extends Exception {
+    public static final String EXCEPTION_FACE = "<OnO>";
```

Comment:
great use of prefixes here to make the constants easily readable!

Reply:
Thank you!